sympy/ntheory/generate.py

```diff
@@ -438,7 +438,7 @@ def prime(nth):
     from sympy.functions.elementary.exponential import log
     from sympy.functions.special.error_functions import li
     a = 2 # Lower bound for binary search
-    b = int(n*(log(n) + log(log(n)))) # Upper bound for the search.
```

Comment:
I don't think we need to make any fixes here.

Reply:
Thanks - I made that change locally but didn't copy to here.